Yoast/wordpress-seo
# Add tests for person schema generator

## Context


* We want the test coverage to be a 100%!

## Summary


This PR can be summarized in the following changelog entry:

* Adds tests for the person schema generator.

## Relevant technical choices:

* Added safety checks around the filters.

## Test instructions

This PR can be tested by following these steps:

* Travis tests should pass.
* The person schema (and thus author schema) should still be the same:
  * Set the site representation to a person. SEO --> Search Appearance --> General --> Knowledge Graph & Schema.org
  * Check the schema output on, for example, an author archive page.
  * Verify it is the same on the `feature/indexables-frontend` branch as on this branch.

## UI changes

## Documentation

## Quality assurance


Fixes #
